frontend/src/features/ProducerOrganizationMembership/components/ProducerOrganizationMembershipTable/index.tsx

```diff
@@ -0,0 +1,75 @@
+import { ErrorWall } from '@components/ErrorWall'
+import { BackOfficeBody } from '@features/BackOffice/components/BackofficeBody'
+import { BackOfficeTitle } from '@features/BackOffice/components/BackOfficeTitle'
+import { useGetProducerOrganizationMembershipsQuery } from '@features/ProducerOrganizationMembership/apis'
+import { useBackofficeAppSelector } from '@hooks/useBackofficeAppSelector'
+import { DisplayedErrorKey } from '@libs/DisplayedError/constants'
+import { Accent, Button, DataTable, pluralize } from '@mtes-mct/monitor-ui'
+import Fuse from 'fuse.js'
+import { useMemo, useState } from 'react'
+import styled from 'styled-components'
+
+import { DEFAULT_ROWS_DISPLAYED, MEMBERSHIP_SEARCH_OPTIONS, TABLE_COLUMNS } from './constants'
+import { FilterBar } from './FilterBar'
+
+export function ProducerOrganizationMembershipTable() {
+  const searchQuery = useBackofficeAppSelector(store => store.producerOrganizationMembership.searchQuery)
+  const displayedError = useBackofficeAppSelector(
+    state => state.displayedError[DisplayedErrorKey.BACKOFFICE_PRODUCER_ORGANIZATION_ERROR]
+  )
+  const [rowsDisplayed, setRowsDisplayed] = useState<number>(DEFAULT_ROWS_DISPLAYED)
+
+  const { data: memberships = [] } = useGetProducerOrganizationMembershipsQuery()
+
+  const fuse = useMemo(
+    () => (memberships ? new Fuse(memberships, MEMBERSHIP_SEARCH_OPTIONS) : undefined),
+    [memberships]
+  )
+
+  const filteredMemberships =
+    !!searchQuery?.length && fuse ? fuse.search(searchQuery).map(result => result.item) : memberships
+  const displayedMemberships = filteredMemberships.slice(0, rowsDisplayed)
+
+  if (displayedError) {
+    return (
+      <BackOfficeBody>
+        <ErrorWall displayedErrorKey={DisplayedErrorKey.BACKOFFICE_PRODUCER_ORGANIZATION_ERROR} isAbsolute />
+      </BackOfficeBody>
+    )
+  }
+
+  return (
+    <BackOfficeBody>
+      <BackOfficeTitle>Adhésions aux Organisations de Producteurs (OP)</BackOfficeTitle>
+
+      <FilterBar />
+
+      <TableLegend data-cy="producer_organization_memberships_results">
+        {filteredMemberships.length} {pluralize('résultat', filteredMemberships.length)}
+      </TableLegend>
+      <DataTable
+        columns={TABLE_COLUMNS}
+        data={displayedMemberships}
+        initialSorting={[{ desc: false, id: 'joiningDate' }]}
```

Comment:
```suggestion
        initialSorting={[{ desc: false, id: 'joiningDate' }]}
        tableOptions={{
          getRowId: row => row.internalReferenceNumber
        }}
```

pour éviter les IDs par défaut (= index) qui font entre autre les "scroll back to top" en cas de polling avec changement. Je le corrige en committant sur ta PR ^^.